torch/_dynamo/variables/tensor.py

```diff
@@ -264,7 +264,10 @@ def call_method(
             unimplemented("dynamic Tensor.repeat")
         elif name in ("tolist", "numpy", "backward", "data_ptr"):
             unimplemented(f"Tensor.{name}")
-        elif name == "nonzero" and not config.dynamic_shapes:
+        elif (
+            name in ("nonzero", "argwhere", "unique", "unique_consecutive")
```

Comment:
it would be nice if we could use tags here, like how FakeTensor does it (`Tag.data_dependent_output`). That might not be possible though given that tags exist at the ATen level, and we don't have a version for `torch.*` :(

Reply:
Yeah, you can't use native_functions.yaml tags here, sorry.

Reply:
You might be able to use `operator_schemas.normalize_function` for this purpose